config/initializers/rack_attack.rb

```diff
@@ -12,7 +10,7 @@
   redis_url = configuration_service.redis_uris.to_a[0][1]
   Rack::Attack.cache.store = ActiveSupport::Cache::RedisCacheStore.new(url: redis_url)
 else
```

Comment:
Does the `elsif Rails.env.review?` clause still need to be separate?

Reply:
Ah, good spot, no it doesn't!